miniwallet/contracts/MiniWallet_v2.sol

```diff
@@ -0,0 +1,615 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
what's the difference between v1 and v2? Seems to be just a name change. If it is just to have different namespace (in hardhat), can this contract just inherit v1?

Reply:
Yes v2 is just used for testing the Upgrade workflow.
Have cleaned this up 🙏